sympy/core/exprtools.py

```diff
@@ -1299,7 +1299,9 @@ def factor_nc(expr):
     """Return the factored form of ``expr`` while handling non-commutative
     expressions.
 
-    **examples**
```

Comment:
Yeah, or `--------`.  And neither one needs an extra newline before I code block, so I didn't do that automatically.  (This one and the `liealgebra` scripts were my only manual changes.)  So there were lots of places I didn't add a newline.  But this does complicate the regex a little, and may make it more susceptible to false negatives.  So if this is converted to a code-quality check, it may be better to just add those newlines.